### PR TITLE
update applicationpopup to include images in footer

### PR DESCRIPTION
**Pull request by noodlemoodle**

### CODE REVIEW

File: src/foam/u2/dialog/ApplicationPopup.js [hunk 1]
```diff
@@ -389,18 +394,27 @@ foam.CLASS({
...
               // link
               .start().addClass(self.myClass('footer-center'))
-                .start(self.footerLink ? 'a' : '')
-                  .show(self.footerString$)
-                  .enableClass(this.myClass('footer-link'), this.footerLink$)
-                  .add(self.footerString$)
+                .start(self.footerLink ? 'a' : 'p')
+                  .show(self.footerStrings$)
+                  .enableClass(self.myClass('footer-link'), self.footerLink$)
+                  .forEach(self.footerStrings, function(str) {
```
Comment (sarthak-marwaha) [inline src/foam/u2/dialog/ApplicationPopup.js]: Can we just use htmlView here?

**Comment by kgrgreer** [comment]
LGTM

File: src/foam/u2/dialog/ApplicationPopup.js (hunk 1 above)
Comment (sarthak-marwaha) [reply src/foam/u2/dialog/ApplicationPopup.js]: Will allow for any sort of formatted view we might want

Comment (sarthak-marwaha) [reply src/foam/u2/dialog/ApplicationPopup.js]: https://github.com/kgrgreer/foam3/blob/451768a68acfbebc1167cfe1cb0acdcd8632c44a/src/foam/u2/wizard/views/FocusWizardView.js#L87-L93

File: src/foam/u2/dialog/ApplicationPopup.js [hunk 2]
```diff
@@ -389,21 +390,18 @@ foam.CLASS({
...
-                  .enableClass(this.myClass('footer-link'), this.footerLink$)
-                  .add(self.footerString$)
-                  .attrs({ href: self.footerLink, target: '_blank' })
-                .end()
-              .end()
+              .add(self.slot(function (footerHTML) { 
+                return this.E().start().addClass(self.myClass('footer-center'))
+                  .tag(foam.u2.HTMLView.create({ nodeName: 'div', data: footerHTML })) 
+                .end();
+              }))
```
Comment (sarthak-marwaha) [inline src/foam/u2/dialog/ApplicationPopup.js]:
```suggestion
              .start().addClass(self.myClass('footer-center'))
                .tag(foam.u2.HTMLView.create({ nodeName: 'div', data$: self.footerHTML$ })) 
              .end()
```

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**